config/settings.yml

```diff
@@ -339,7 +339,7 @@ defaults: &defaults
   - url: https://192.168.50.3:25555
     primary: false        # is true if the director takes part in lifecycle operations except create
     support_create: false  # is true if director is expected to 'create deployments'
-    name: bootstrap-bosh
```

Comment:
Please remve this altogether and have the next two as bosh and bosh-sf.

Reply:
removed.